### DISCUSSION TITLE
Local Variable Type Inference Style

### DISCUSSION THREAD

**Discussion by bmasetto** [checkstyle/checkstyle#10531] [Q&A] [answered]
Hey, guys!

Is there a way to use checkstyle to warn me of local variable type inference possible usages?

https://developer.oracle.com/java/jdk-10-local-variable-type-inference.html
http://openjdk.java.net/projects/amber/LVTIstyle.html

Thanks!!!

**Comment by strkkk** [comment, marked as answer]
There is a similar ticket https://github.com/checkstyle/checkstyle/issues/8978

About your question - there is no such check at the moment, you are welcome to submit proposals
